src/Dashboard/Data/FunctionInstanceSnapshot.cs

```diff
@@ -17,7 +17,7 @@ public class FunctionInstanceSnapshot
 
         public Guid Id { get; set; }
 
-        public Guid HostInstanceId { get; set; }
```

Comment:
Based on this code, I suspect that it would break something. https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Dashboard/Data/FunctionInstanceSnapshot.cs#L20

@MikeStall -- how would I verify that this stuff works if we change this from Guid to string?